_schemaV2.graphql

```diff
@@ -1379,6 +1380,7 @@ type Artwork implements Node & Searchable & Sellable {
   ): Partner
   pickupAvailable: Boolean
   priceCurrency: String
+  priceDisplay: String
```

Comment:
As mentioned offline we should not use `priceDisplay` name for this because we already use this term in CMS for determining how we should display the price that users entered. It's a combination of 'price_hidden' and 'display_price_range' fields in gravity and has values of 'exact' or 'range' or 'hidden' in CMS.